src/components/QRCode/index.js

```diff
@@ -10,9 +10,9 @@ const propTypes = {
     url: PropTypes.string.isRequired,
     /**
      * The logo which will be displayed in the middle of the QR code.
-     * Follows `ImageSourcePropType` from react-native.
+     * Follows ImageProps href from react-native-svg that used by react-native-qrcode-svg.
```

Comment:
```suggestion
     * Follows ImageProps href from react-native-svg that is used by react-native-qrcode-svg.
```

Small grammar mistake.